pennylane/__init__.py

```diff
@@ -53,6 +53,7 @@
 import pennylane.qnode_old
 from pennylane.transforms import (
     adjoint,
+    adjoint_metric_tensor,
```

Comment:
I imagine this could become confusing 😆

Reply:
Oh no, naming issues! :D

Reply:
there was a concerted internal effort to rename the adjoint diff method the 'rewind diff method' a while back 😆

Reply:
I must admit, it seems that this did not reach me :D I like `rewind_metric_tensor` :)